tests/unit/core/test_installAddon.js

```diff
@@ -261,14 +271,18 @@ describe(__filename, () => {
   });
 
   it('sets the current status in componentDidMount with an addonManager', () => {
+    //TODO: I'm having issues with this test.
+    // mapStateToProps doesn't seem to be called, so there's no currentVersion.
```

Comment:
You probably already knows this :) but I think if you rebase now, looks like this might be fixed (per #6894)

Reply:
Yes, thanks!